softwarereview_editor_management.Rmd

```diff
@@ -23,10 +23,9 @@ Steps:
 
 ## Inviting a new editor {#inviting-a-new-editor}
```

Comment:
Note for future PRs, we try to have a single language reviewed first before we add translations.